src/main/java/runwar/Start.java

```diff
@@ -120,6 +120,7 @@ public void handleRequest(final HttpServerExchange exchange) throws Exception {
 
             Server server = new Server();
             try {
+                RunwarLogger.LOG.info("test bal here startServer /*/*/*/*...");
```

Comment:
@BalbinoOrtus Need to remove this debugging code?

Reply:
snap totally right, on it